components/PlantCard/styles.ts

```diff
@@ -0,0 +1,97 @@
+import styled from 'styled-components';
+
+export const Card = styled.div`
+  position: relative;
+  height: 40vh;
+  width: 20vw;
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  border-radius: 12px;
+  background-color: white;
+  box-shadow:
+    0 24px 38px 3px rgba(0, 0, 0, 0.14),
+    0 9px 46px 8px rgba(0, 0, 0, 0.12),
+    0 11px 15px -7px rgba(0, 0, 0, 0.2);
+
+  &.greenBorder {
+    border: 1px solid #95b509;
+  }
+`;
+
+export const CardPic = styled.div`
+  height: 60%;
+  width: 100%;
+  background-color: #f5f6f6;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-top-left-radius: 12px;
+  border-top-right-radius: 12px;
+`;
+
+export const CardContent = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding-left: 1vw;
+  height: 40%;
+  row-gap: 1vh;
+
+  > * {
+    margin: 0;
+  }
+
+  > h2 {
+    font-size: 1.5vw;
+    margin-top: 1vh;
+  }
+`;
+
+export const PlantAttributes = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 1vh;
+`;
+
+export const Attribute = styled.div`
+  display: flex;
+  flex-direction: row;
+
+  > * {
+    margin: 0;
+    font-size: 1vw;
```

Comment:
let's use rem for font-sizes! 
check figma for font-sizes and use the conversion: 16px = 1 rem